pkg/genall/genall.go

```diff
@@ -101,6 +102,8 @@ type Runtime struct {
 	OutputRules OutputRules
 	// ErrorWriter defines where to write error messages.
 	ErrorWriter io.Writer
+	// Verbose enables verbose logging for debugging purposes.
+	Verbose bool
```

Comment:
For future expansion of options here, a bool is probably not the best choice. Typically you might see an enum here of Error, Warn, Info and Debug, what does slog expose as levels? Should we mirror their option?